app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -616,6 +616,16 @@ export default Ember.Component.extend({
     Ember.run.scheduleOnce("afterRender", () => $textarea.focus());
   },
 
+  paste: function(e) {
+    var items = (e.clipboardData || e.originalEvent.clipboardData).items;
```

Comment:
Note that we prefer `let` instead of `var` because 

> let allows you to declare variables that are limited in scope to the block, statement, or expression on which it is used. This is unlike the var keyword, which defines a variable globally, or locally to an entire function regardless of block scope.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let